addons/api/mirage/scenarios/default.js

```diff
@@ -68,9 +68,11 @@ export default function (server) {
     const oidcAuthMethod = globalAuthMethods.filter(
       (auth) => auth.type === 'oidc'
```

Comment:
this can be replaced with the imported type constant

Reply:
@DhariniJeeva yes, good catch

Reply:
@DhariniJeeva added